### ISSUE TITLE
Is this library still maintained and will there be any new release?

### ISSUE THREAD

**Issue by dcgavril** [satyan/sugar#348]
From what I can see, there are commits but none by the owner (except for a merge), and nobody did any release updates in last year. So, will it be better from now on? will there be a release soon? (.jar in the repository - easier to use and reassuring for users)
It's very important to know this, if you're trying to integrate this library.
Thanks!

**Comment by satyan** [comment]
Yes, there will be new releases and will be maintained better going forward. I'm trying to devote more time for the project and would be taking active part in the maintenance. 

Thanks for the query.

**Comment by dcgavril** [comment]
Thanks for the reply, good luck with the updates!

**Comment by black-snow** [comment]
This issue can be closed

**Comment by black-snow** [comment]
It'd be nice to have at least a 1.3.2 with bugfixes.
